pkg/syncthing/syncthing.go

```diff
@@ -183,7 +207,13 @@ func (s *Syncthing) StartLocalSyncthing() error {
 func (s *Syncthing) Ping() (bool, error) {
 	_, err := s.Client.SendRequest(GET, "/rest/system/ping", nil, nil)
 	if err != nil {
-		logrus.Debug("Failed to ping syncthing: ", err)
+		logrus.WithFields(logrus.Fields{
+			"name":    s.Name,
+			"port":    s.Port,
+			"homedir": s.HomeDirectory,
+			"apikey":  s.ApiKey,
+			"config":  s.Config,
+		}).Debug("Failed to ping syncthing: ", err)
```

Comment:
with error

Reply:
Sorry, I see. I will fix it in the next commit.